src/scss/components/button/_mixins.scss

```diff
@@ -120,16 +126,18 @@
   }
 }
 
-@mixin button-small {
+/// ---------- BUTTON SIZES ----------
+
+@mixin button-small { // less: #Button.small()
   padding: 0.125rem 0.5rem; // 2px 8px
 }
 
-@mixin button-medium {
+@mixin button-medium { // less: #Button.medium()
   font-size: 0.875rem;
-  padding: 8px 12px;
+  padding: 0.5rem 0.75rem; // 8px 12px;
```

Comment:
Avoiding the use of hard `px` values and to match how padding is defined in the other sizes.